other/tips/replacer/match-and-replace/README.md

```diff
@@ -22,14 +24,71 @@ In ZAP these can be Revealed with standard functionality: <https://www.zaproxy.o
 
 ![](images/show-hidden-1.png)
 
+<details>
+<summary>Show Hidden UI Elements</summary>
+
+```js
+// This script adds a Replacer rule
+var Control = Java.type("org.parosproxy.paros.control.Control");
+var extReplacer = Control.getSingleton().getExtensionLoader().getExtension("ExtensionReplacer");
```

Comment:
Use `control` instead of `Control.getSingleton()`, the import of `Control` should not be needed.